src/components/FeedPage.vue

```diff
@@ -18,8 +18,8 @@
     </span>
 
     <span class="uk-align-right">
-        {{ $t("actions.sort_by") }}
-        <select class="uk-select uk-width-auto" v-model="selectedSort" @change="onChange()">
+        <label for='ddlSortBy'>{{ $t("actions.sort_by") }}</label>
```

Comment:
You've used a mixture of single quotes and double-quotes. You should be using only double quotes. If you are using prettier, there is a config that you can import to do this when formatting :)

Reply:
Sorry about that, should be fixed now